skymp5-front/src/components/SkyrimButton/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './styles.scss';
```

Comment:
`index.tsx` and `style.scss` becomes `SkyrimButton.tsx` and `SkyrimButton.scss` accordingly.